### ISSUE TITLE
DOC: Microsoft Word on Windows unable to open MHT output, treats it as unknown HTML file

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/metanorma-standoc#799]
As reported (and demonstrated) by @stuartgalt in https://github.com/metanorma/iso-10303-2/issues/316, Microsoft Word is unable to recognize the Metanorma Word output (an MHT file) at all regardless of the file extension used (`.doc` and `.mht`). 

The same file opens perfectly using Microsoft Word on Mac.

On loading the file, the user is prompted with a modal popup box requesting the user to select the type of file it is, providing close to or more than 10 different options such as "Single Page HTML File" (which was the only one that made sense).

When selecting "Single Page HTML File", Word will take a long time to process the document, just like when loading a normal HTML file. Once done, it seems that the "Word content" is newly formatted and the embedded Word styles are all gone.

This is a huge problem for Windows users. I'm not sure how we can address this using https://github.com/metanorma/html2doc .

**Comment by opoudjis** [comment]
The only difference between .DOC out of the factory and MHT is that the DOC is zipped. The user who reported it's "just an HTML file" is wrong about that, it is a MIME package. I will need to investigate, I am not convinced the issue has been diagnosed correctly.

**Comment by ronaldtse** [comment]
@opoudjis no, I saw with my own eyes the entire flow on Windows Word. The file cannot be recognized by Word as a Word file.

**Comment by ronaldtse** [comment]
It might not be an ecosystem-wide issue but it is certainly an issue with Windows Word.

**Comment by opoudjis** [comment]
I have just opened a document I have just generated, on Windows 10, Word for Office 365 MSO, Version 2307, Build 16.0.16626.20086.

This makes this a bug I cannot address, because I cannot replicate it. Word on opening the document (on a PC that has never opened a Metanorma document before) did say "Do you want to make Word your default Web browser", which means it is seeing MHT as HTML, but all styling is preserved.

I require from @stuartgalt details of his Office setup, and confirmation that he's getting the same behaviour on the attached document, before we can do anything further about this. 

[a.doc.zip](https://github.com/metanorma/metanorma-standoc/files/12321503/a.doc.zip)

We also need to be asking our users how widespread an issue this is.
